src/chrome/offscreen/wallet-connection/factory.ts

```diff
@@ -0,0 +1,47 @@
+import { AppLogger } from 'utils/logger'
+import { WalletConnectionClient } from './wallet-connection-client'
+import { config, radixConnectConfig } from 'config'
+import { MessagesRouter } from 'chrome/offscreen/wallet-connection/messages-router'
+import { ConnectorClient } from '@radixdlt/radix-connect-webrtc'
+import { syncClient } from './sync-client'
+
+export type walletConnectionClientFactory = typeof walletConnectionClientFactory
+
+export const walletConnectionClientFactory = (input: {
```

Comment:
follow the same name convention as the other clients. `WalletConnectionClientFactory`